src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -0,0 +1,16 @@
+package cromwell.engine.backend
+
+import cromwell.binding.values.WdlValue
+import cromwell.binding.{Call, WdlStandardLibraryFunctions, WorkflowDescriptor}
+
+import scala.util.Try
+
+trait BackendCall {
+  val backend: Backend
```

Comment:
abstract entities in traits should be defs unless there's a strong reason not to do it.